hasktorch/src/Torch/NN.hs

```diff
@@ -163,8 +163,9 @@ conv2d'' layer stride padding input =
 
 instance Randomizable Conv2dSpec Conv2d where
   sample Conv2dSpec{..} = do
-      w <- makeIndependent =<< randnIO' [outputChannelSize, inputChannelSize, kernelSize0, kernelSize1]
-      b <- makeIndependent =<< randnIO' [outputChannelSize]
+      w <- makeIndependent =<< kaimingUniform FanIn (LeakyRelu $ Prelude.sqrt 5.0) [outputChannelSize, inputChannelSize, kernelSize0, kernelSize1]
+      uniform <- randIO' [outputChannelSize]
+      b <- makeIndependent =<< pure (mulScalar ((2::Int) * kernelSize0 * kernelSize1) (subScalar (0.5 :: Float) uniform))
```

Comment:
@apaszke Is this initialization fine?

Reply:
Weight looks good, but bias seems to be missing a square root?